File: expresslk-webportal/module/Api/src/Api/Client/Soap/Core/HoldCriteria.php
<?php

namespace Api\Client\Soap\Core;

class HoldCriteria
{

    /**
     * @var int $boardingPoint
     * @access public
     */
    public $boardingPoint = null;

    /**
     * @var int $droppingPoint
     * @access public
     */
    public $droppingPoint = null;

    /**
     * @var string $resultIndex
     * @access public
     */
    public $resultIndex = null;

    /**
     * @var SeatCriteria[] $seatCriterias
     * @access public
     */
    public $seatCriterias = null;

    /**
     * @param int $boardingPoint
     * @param int $droppingPoint
     * @param string $resultIndex
     * @access public
     */
    public function __construct($boardingPoint, $droppingPoint, $resultIndex)
    {
      $this->boardingPoint = $boardingPoint;
      $this->droppingPoint = $droppingPoint;
      $this->resultIndex = $resultIndex;
    }

}

File: expresslk-services/ExpressLK-core/src/main/java/lk/express/HoldCriteria.java
package lk.express;

import java.util.List;

import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;

@XmlRootElement
@XmlType(name = "HoldCriteria", namespace = "http://express.lk")
public class HoldCriteria {

	private String resultIndex;
	private List<SeatCriteria> seatCriterias;

	private int boardingPoint;
	private int droppingPoint;

	public int getBoardingPoint() {
		return boardingPoint;
	}

	public void setBoardingPoint(int boardingPoint) {
		this.boardingPoint = boardingPoint;
	}

	public int getDroppingPoint() {
		return droppingPoint;
	}

	public void setDroppingPoint(int droppingPoint) {
		this.droppingPoint = droppingPoint;
	}

	public List<SeatCriteria> getSeatCriterias() {
		return seatCriterias;
	}

	public void setSeatCriterias(List<SeatCriteria> seatCriterias) {
		this.seatCriterias = seatCriterias;
	}

	public String getResultIndex() {
		return resultIndex;
	}

	public void setResultIndex(String resultIndex) {
		this.resultIndex = resultIndex;
	}

	@XmlType(name = "SeatCriteria", namespace = "http://express.lk")
	public static class SeatCriteria {

		private int sectorNumber;
		private List<String> seats;

		public int getSectorNumber() {
			return sectorNumber;
		}

		public void setSectorNumber(int sectorNumber) {
			this.sectorNumber = sectorNumber;
		}

		public List<String> getSeats() {
			return seats;
		}

		public void setSeats(List<String> seats) {
			this.seats = seats;
		}
	}
}
